### DISCUSSION TITLE
Fingerprint sensor / Biometric authentication

### DISCUSSION THREAD

**Discussion by LuisAlfredo92** [unoplatform/uno#14938] [Q&A] [answered]
Is there any way to use the fingerprint sensor?
I found about https://github.com/unoplatform/uno/discussions/10914, but the answers there use Xamarin, and now it isn't supported by Uno

So is there any other way besides using native code? Or are those packages still optimal?

**Comment by limeniye** [comment, marked as answer]
Hello, @LuisAlfredo92!
Try to do the following steps:

- Define your code as code assigned to the Android platform: ```#if ANDROID *Your code* #endif```
- Change the target platform to Android from the top of the screen
- Add the missing usings

![image](https://github.com/unoplatform/uno/assets/69314237/5cd01cfe-699a-4afe-906f-4f681298f1c8)

**Reply by LuisAlfredo92** [reply]
Oh okay, so I'll go with native code
Thanks!

**Reply by jeromelaban** [reply]
You can also try https://github.com/nventive/BiometryService/tree/main, which abstracts some of those for iOS/Android.
